code/__DEFINES/sound.dm

```diff
@@ -1,3 +1,4 @@
 //max channel is 1024. Only go lower from here, because byond tends to pick the first availiable channel to play sounds on
 #define CHANNEL_LOBBYMUSIC 1024
 #define CHANNEL_ADMIN 1023
+#define CHANNEL_VOX 1022
```

Comment:
@Fox-McCloud look at the ```open_sound_channel()``` we went with at the end, we loop through form 1 to HIGHEST_OPEN_CHANNEL, there's no collisions.

the issue with leaving it at 0 is that byond doesn't know we've ""reserved"" 1022, 1023, 1024 etc, so it can cut out those sounds, YES it picks the lowest first, but we've had sound cutout issues (which are caused by channel overlaps) so we're basically just saying "fine, overlap if you want, but don't pick X, Y or Z channel"